Add note about upload speed error and fix (#2224)
astronomer/docs: astro/create-first-dag.md

After you select a Deployment, the CLI parses your DAGs to ensure that they don't contain basic syntax and import errors. If your code passes the parse, the Astro CLI builds all files in your Astro project directory into a new Docker image and then pushes the image to your Deployment on Astro. 

> If you have a slow upload speed with your internet connection, it can cause your Astro project deploy to fail. You can fix this by switching to [DAG-only deploys](deploy-code.md#enable-dag-only-deploys-on-a-deployment), where the Astro CLI only pushes your DAGs to your Deployment instead of a new Docker image.

Reviewer: ```suggestion
If your internet connection has slow upload speeds, it can cause your Astro project deploy to fail. You can fix this by switching to [DAG-only deploys](deploy-code.md#enable-dag-only-deploys-on-a-deployment), where the Astro CLI only pushes your DAGs to your Deployment instead of a new Docker image.
```
Two things:

- Should we put this in an `info` box since it's optional troubleshooting information?
- I'm not sure how this would resolve the circumstance in this quickstart. You always need to deploy a complete project as your first deploy - DAG-only deploys will only work once you've pushed your project up to begin with.
